docs/blog/2019-02-06-government-open-data-site-with-gatsby/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: "How we're migrating a government open data site to Gatsby"
+date: 2019-02-05
```

Comment:
Let's schedule this for Friday

```suggestion
date: 2019-02-08
```


(will also need to re-name the file if that's OK with you both!)

Reply:
No problem.